platform_plugin_aspects/extensions/filters.py

```diff
@@ -42,11 +42,17 @@ def run_filter(
 
         user = get_current_user()
 
+        language = get_model("user_preference").get_value(user, "pref-lang") or "en_US"
+        formatted_language = language.replace("-", "_")
+        if formatted_language not in settings.SUPERSET_DASHBOARD_LOCALES:
+            formatted_language = None
```

Comment:
If we're defaulting to `en_US` above, why not use that here instead of `None`? Not sure what `None` does here.

Reply:
It uses the original dashboard instead of the `en_US` version but you are right